features.json

```diff
@@ -103,65 +138,82 @@
+				"jsStringBuiltins": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
 				"memory64": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
 				"multiMemory": "120",
 				"multiValue": "85",
 				"mutableGlobals": "74",
-				"relaxedSimd": "114",
 				"referenceTypes": "96",
+				"relaxedSimd": "114",
 				"saturatedFloatToInt": "75",
 				"signExtensions": "74",
 				"simd": "91",
 				"tailCall": "112",
 				"threads": "74",
-				"typeReflection": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"]
+				"typedFunctionReferences": "119",
+				"typeReflection": ["flag", "Requires flag `chrome://flags/#enable-experimental-webassembly-features`"],
+				"webContentSecurityPolicy": "97"
 			}
 		},
 		"Firefox": {
 			"url": "https://www.mozilla.org/firefox/",
 			"logo": "/images/firefox.svg",
 			"features": {
 				"bigInt": "78",
+				"branchHinting": ["flag", "Enabled in Nightly, requires flag `javascript.options.wasm_branch_hinting` in Beta/Release"],
 				"bulkMemory": "79",
-				"exceptions": "100",
+				"customAnnotationSyntaxInTheTextFormat": null,
+				"exceptions": ["flag", "Enabled in Nightly, requires flag `javascript.options.wasm_exnref` in Beta/Release. Outdated version of proposal available since 100"],
 				"extendedConst": "112",
 				"gc": "120",
-				"jspi": ["flag", "Requires flag `javascript.options.wasm_js_promise_integration` in Nightly, unavailable in Beta/Release"],
-				"memory64": ["flag", "Enabled in Nightly, unavailable in Beta/Release"],
+				"jspi": ["flag", "Requires flag `javascript.options.wasm_js_promise_integration`"],
+				"jsStringBuiltins": ["flag", "Requires flag `javascript.options.wasm_js_string_builtins`"],
+				"memory64": ["flag", "Enabled in Nightly, requires flag `javascript.options.wasm_memory64` in Beta/Release"],
 				"multiMemory": "125",
 				"multiValue": "78",
 				"mutableGlobals": "61",
 				"referenceTypes": "79",
-				"relaxedSimd": ["flag", "Enabled in Nightly, unavailable in Beta/Release"],
+				"relaxedSimd": ["flag", "Enabled in Nightly, requires flag `javascript.options.wasm_relaxed_simd` in Beta/Release"],
 				"saturatedFloatToInt": "64",
 				"signExtensions": "62",
 				"simd": "89",
 				"tailCall": "121",
 				"threads": "79",
-				"typeReflection": ["flag", "Enabled in Nightly, unavailable in Beta/Release"]
+				"typedFunctionReferences": "120",
+				"typeReflection": ["flag", "Requires flag `--enable-wasm-type-reflections`"],
+				"webContentSecurityPolicy": "102"
 			}
 		},
 		"Safari": {
 			"url": "https://www.apple.com/safari/",
 			"logo": "/images/safari.svg",
 			"features": {
 				"bigInt": ["15", "wasm-bigint is supported in desktop Safari since 14.1 and iOS Safari since 14.5; however BigInt64Array, which is needed by Emscripten, was released in 15"],
+				"branchHinting": "16",
```

Comment:
It has been added in Safari 16, and was immediately enabled by default. There is now no longer any feature to toggle it. I'll recheck to see if they "removed the feature" or if that's just because they don't need a toggle anymore.

Reply:
Yep, they removed the flag because it's "already shipped" here: https://github.com/WebKit/WebKit/commit/ca9f4e631e5eaf44ce22e1f5cc29519be9488825

Reply:
aha, and it was enabled in https://github.com/WebKit/WebKit/commit/e1f19d73022eeb9d97b02fd64ba45e40d346d169
I'll bet this was an accident, actually. TC39 has a similar stage process for JS features, and IIRC browsers typically ship at stage 3. Anyway, this is fine for the website, I'll follow up and make sure there's nothing that they need to fix or update with the proposal :)